be/src/olap/tablet.cpp

```diff
@@ -88,6 +88,10 @@ OLAPStatus Tablet::_init_once_action() {
             << ", version_size=" << _tablet_meta->version_count();
 
     // init cumulative compaction policy by type
+    if (_cumulative_compaction_type == "") {
```

Comment:
Now _cumulative_compaction_policy will be created in _calc_cumulative_compaction_score, is it needed to create it here initially?

Reply:
> Now _cumulative_compaction_policy will be created in _calc_cumulative_compaction_score, is it needed to create it here initially?

I also konw that we don't need to execute`create_cumulative_compaction_policy()`here, but UT will have some problems if we remove it. Next, I will refacor UT.